src/ui/language_region_settings_page.ui

```diff
@@ -0,0 +1,122 @@
+    <property name="margin-start">20</property>
+    <property name="margin-end">10</property>
+    <property name="margin-top">10</property>
+    <property name="margin-bottom">10</property>
+    <child>
+      <object class="GtkLabel">
+      <property name="label">Language &amp; Region</property>
+      <style><class name="title"/></style>
+      <property name="margin-end">10</property>
+      <property name="margin-top">10</property>
+      <property name="margin-bottom">10</property>
+      <property name="halign">start</property>
+      </object>
+    </child>
+    <child>
+      <object class="GtkBox">
+      <style><class name="info-box"/></style>
+      <property name="orientation">vertical</property>
+      <property name="vexpand">false</property>
+      <property name="hexpand">true</property>
+      <property name="margin-top">10</property>
+      <property name="margin-bottom">10</property>
+      <property name="spacing">10</property>
+        <child>
+          <object class="GtkLabel">
+          <style><class name="normal-text"/></style>
+          <property name="label">Language</property>
+          <property name="halign">start</property>
+          </object>
+        </child>
+        <child>
+          <object class="GtkDropDown" id="language_switch">
+            <style><class name="dropdown-button"/></style>
+            <property name="halign">start</property>
+            <property name="valign">center</property>
+            <property name="width-request">220</property>
+            <property name="height-request">30</property>
+	    <property name="expression">
+              <lookup name="display" type="DataGObject"/>
+	    </property>
+          </object>
+        </child>
+      </object>
+    </child>
+    <child>
+      <object class="GtkBox">
+      <style><class name="info-box"/></style>
+      <property name="orientation">vertical</property>
+      <property name="vexpand">false</property>
+      <property name="hexpand">true</property>
+      <property name="margin-top">10</property>
+      <property name="margin-bottom">10</property>
+      <property name="spacing">10</property>
+        <child>
+          <object class="GtkLabel">
+          <style><class name="normal-text"/></style>
+          <property name="label">Timezone</property>
+          <property name="halign">start</property>
+          </object>
+        </child>
+        <child>
+          <object class="GtkDropDown" id="timezone_switch">
+            <style><class name="dropdown-button"/></style>
+            <property name="halign">start</property>
+            <property name="valign">center</property>
+            <property name="width-request">220</property>
+            <property name="height-request">30</property>
+	    <property name="expression">
+              <lookup name="display" type="DataGObject"/>
+	    </property>
+          </object>
+        </child>
+      </object>
+    </child>
```

Comment:
add placeholder, so that footer will be on the bottom:
```
<child>
      <object class="GtkBox">
      <property name="vexpand">true</property>
      </object>
</child>
```

Reply:
Made the last item before footer expand instead, no need for a placeholder.